docs/docs/optimizing-performance.md

```diff
@@ -232,3 +232,25 @@ In this case, since a new reference is returned when mutating `x`, we can safely
 Two other libraries that can help use immutable data are [seamless-immutable](https://github.com/rtfeldman/seamless-immutable) and [immutability-helper](https://github.com/kolodny/immutability-helper).
 
 Immutable data structures provide you with a cheap way to track changes on objects, which is all we need to implement `shouldComponentUpdate`. This can often provide you with a nice performance boost.
+
+## Profiling Components with Chrome Timeline
+
+You can now visualize React components in the Chrome Timeline. This lets you see which components exactly get mounted, updated, and unmounted, how much time they take relative to each other.
```

Comment:
"can now" is good wording for blog post but not very good for static docs.
Maybe instead:

>In development mode, you can visualize how components mount, update, and unmount, using the performance tools in supported browsers. For example, to do this in Chrome: